packages/evolution-generator/src/scripts/generate_widgets.py

```diff
@@ -41,16 +41,20 @@ def convert_excel_to_typescript(section):
             has_input_range_import = False
             has_custom_widgets_import = False
             has_custom_validations_import = False
+            has_custom_conditionals_import = False
             has_help_popup_import = False
             has_persons_cnt_label = False
             has_gendered_suffix_label = False
             for row in section_rows:
                 if row['choices']:
                     has_choices_import = True
-                if row['conditional'].endswith('CustomValidation'):
-                    # Check to see if the validation finish with 'CustomConditional'
+                if row['validation'].endswith('CustomValidation'):
+                    # Check to see if the validation finish with 'CustomValidation'
                     has_custom_validations_import = True
-                if row['conditional']:
+                if row['conditional'] and row['conditional'].endswith('CustomConditional'):
```

Comment:
Dans le cas: if row['conditional'] and row['conditional'].endswith('CustomConditional'): 
Il y a pas aussi has_conditionals_import = True
Car il y a des conditions et il y a des customConditional, il y a les deux cas, non?

Reply:
S'il y a une conditonnelle qui ne termine pas par CustomConditional, c'est dans le elif 2 lignes plus bas. Il ajoute le conditionals_import. Donc si dans une section, il y a les 2, les deux imports seront présents. S'il n'y a que CustomConditional, c'est le custom conditional seulement qui sera importé. Ou est-ce qu'il faudrait importer les condtionnelles même s'il n'y a que des custom conditional?

Reply:
Ah non, tu as raison! :)